integration-tests/known-contracts-PtKathman.ts

```diff
@@ -3,4 +3,4 @@ export const knownBigMapContractPtKathman = "KT1TZWawPbU6mCMa1x6hEZcuZ2RtJxqprsk
 export const knownTzip12BigMapOffChainContractPtKathman = "KT1WePZzvQxLFmb9QfpLK9rgUVcRPHzjQLhN";
 export const knownSaplingContractPtKathman = "KT1Riu8a1dbWRMmFpMzb9PgdJbhHAAHuvDyJ";
 export const knownOnChainViewContractAddressPtKathman = "KT1My3eKdYKW3H27jZ2oVcNCXGNBDBcTuYiR";
-export const txRollupAddressPtKathman = "txr1R5hkRCJjMugm2Z6gmh5J6Q4WJToHu8mBe";
+export const txRollupAddressPtKathman = "txr1R5hkRCJjMugm2Z6gmh5J6Q4WJToHu8mBe";
```

Comment:
this has been removed from master with my pr so it may be a conflict or readded

Reply:
thanks